core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5835,9 +5835,13 @@ components:
       description: ""
     SpendReportConfiguration:
       required:
+        - datasetGoogleProject
```

Comment:
We need to know which Google project the dataset resides in. Because the billing export is configured at the billing account level, the dataset can live in any of the projects attached to that billing account- even ones that Terra doesn't know about.

An alternative to another field is to require users to specify the full dataset ID in the same format that Google uses:

i.e. `project-name:my_dataset_name`